### ISSUE TITLE
Support recursive editing

### ISSUE THREAD

**Issue by jlstevens** [ioam/parambokeh#58]
One feature parambokeh currently lacks before it has full feature parity with paramnb is [Recursive editing](https://github.com/ioam/paramnb/blob/master/doc/AdditionalFeatures.ipynb).

**Comment by jbednar** [comment]
That's an extremely important feature.  I don't know how to do it in Bokeh, but we really need to figure out a way.

**Comment by philippjfr** [comment]
I don't think this would be too hard to implement, you'd just have to insert an extra widget box when you toggle the nested object. The main obstacle here will probably be in bokeh's layout capabilities, it's not quite clear to me what layout construct would let us lay out the nested widgets nicely.
